src/libstd/net/udp.rs

```diff
@@ -48,12 +48,14 @@ use time::Duration;
 /// {
 ///     let mut socket = UdpSocket::bind("127.0.0.1:34254")?;
 ///
-///     // read from the socket
+///     // Read a single message from the socket. If `buf` is too small to hold
+///     // the message, it will be cut off.
```

Comment:
> ```rust
> // Read a single message from the socket. If `buf` is too small to hold
> // the message, it will be cut off.
> let mut buf = [0; 10];
> let (amt, src) = socket.recv_from(&mut buf)?;
>
> // send a reply to the socket we received data from
> let buf = &mut buf[..amt]; // create a slice into the buffer that's
>                            // no longer than the received data
> ```

This feels jarring to me in that it uses a mix of sentence-case and lowercase, punctuation and no punctuation, and full line and end of line comments. Could you tidy this up to make it feel more consistent? Typically I prefer sentence-case, with punctuation, and comments without any code on the same line. But check the rest of this module because if everything else is consistently in a different style, let's follow that.